databases.go

```diff
@@ -904,6 +913,20 @@ func (svc *DatabasesServiceOp) DeletePool(ctx context.Context, databaseID, name
 	return resp, nil
 }
 
+// UpdatePool will update an existing database connection pool
+func (svc *DatabasesServiceOp) UpdatePool(ctx context.Context, databaseID, name string, updatePool *DatabaseUpdatePoolRequest) (*Response, error) {
```

Comment:
We should add checks to the data coming into the function. 

- check if `databaseID` is greater than 1 [example](https://github.com/digitalocean/godo/blob/main/droplets.go#L349-L351)
- check if `name` is not empty [example](https://github.com/digitalocean/godo/blob/main/droplets.go#L444-L446)
- check if `updatePool` is not nil [example](https://github.com/digitalocean/godo/blob/main/droplets.go#L396-L398)

Reply:
sure. Just added checks on Mode, Database, Size, and the `updatePool` struct.